bookkeeper-server/src/test/java/org/apache/bookkeeper/client/TestGetBookieInfoTimeout.java

```diff
@@ -62,8 +65,17 @@ public TestGetBookieInfoTimeout() {
     @Before
     public void setUp() throws Exception {
         super.setUp();
-        channelFactory = new NioClientSocketChannelFactory(Executors.newCachedThreadPool(), Executors
-                .newCachedThreadPool());
+        if (SystemUtils.IS_OS_LINUX) {
+        	try {
+        		eventLoopGroup = new EpollEventLoopGroup();
+        	} catch (Exception e) {
```

Comment:
you MUST  catch (Throwable) and not Exception, as you do in other parts.
Maybe it would be better to have a common utility function to create all the EventLoopGroups

in fact running without the native epoll support libraries on the library path make the test fail